samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Wasm/Uno.Toolkit.Samples.Wasm.csproj

```diff
@@ -42,15 +42,15 @@
 	</ItemGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-		<PackageReference Include="Uno.Cupertino" Version="1.2.0" />
+		<PackageReference Include="Uno.Cupertino" Version="1.4.0-dev.17" />
 		<PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.3.0-dev.1" />
-		<PackageReference Include="Uno.Material" Version="1.2.0" />
-		<PackageReference Include="Uno.UI.WebAssembly" Version="4.0.13" />
-		<PackageReference Include="Uno.UI.RemoteControl" Version="4.0.13" Condition="'$(Configuration)'=='Debug'" />
-		<PackageReference Include="Uno.Wasm.Bootstrap" Version="3.2.0" />
-		<PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="3.2.0" />
+		<PackageReference Include="Uno.Material" Version="1.4.0-dev.17" />
+		<PackageReference Include="Uno.UI.WebAssembly" Version="4.2.0-dev.513" />
+		<PackageReference Include="Uno.UI.RemoteControl" Version="4.2.0-dev.513" Condition="'$(Configuration)'=='Debug'" />
+		<PackageReference Include="Uno.Wasm.Bootstrap" Version="4.0.0-dev.101" />
+		<PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="4.0.0-dev.101" />
```

Comment:
Don't update yet, this will fail on CI.

Reply:
do we have an issue or an eta for this?

Reply:
uno.ui package update will come from #204